Transfer/Sources/Extensions/TransferThemeManager.swift

```diff
@@ -55,29 +55,29 @@ public extension ThemeManager {
 
     private static func applyToTransferAllFundsCell() {
         let proxy = TransferAllFundsCell.appearance()
-        proxy.titleLabelFont = Theme.Label.bodyFontMedium
+        proxy.titleLabelFont = Theme.Label.bodyFont
         proxy.titleLabelColor = Theme.Label.color
     }
 
     private static func applyToTransferAmountCell() {
         let proxy = TransferAmountCell.appearance()
-        proxy.titleLabelFont = Theme.Label.bodyFontMedium
+        proxy.titleLabelFont = Theme.Label.captionOne
```

Comment:
Why did you change from bodyFontMedium to captionOne? Change it to bodyFont

Reply:
Changed